### DISCUSSION TITLE
Github Mobile iOS: Secondary Rate Limits violated

### DISCUSSION THREAD

**Discussion by badcel** [community/community#7894] [Mobile] [answered]
Initial report can be found here: https://github.community/t/github-mobile-app-on-ios-secondary-rate-limits-violated/210094

My condensed findings regarding the violoated secondary rate limits are:
 - I can not reproduce it on android with the GitHub App
 - I can reproduce it on iOS with the GitHub App (1.40 on iOS 15.1). Happens in under one minute while browing my repository.
 - I can not reproduce it on the web interface.
 - It seems only to happen on my own organizations repository: https://github.com/gircore/gir.core
 - On the repository the dependency graph size was [increased manually](https://github.community/t/dependency-graph-manifest-files-limit/133284/52). It was stated that it is probably not related.
 - I did log out and reinstall the app and logged in again.

<img src="https://user-images.githubusercontent.com/1218031/142572353-81f40899-8e7f-488b-b99f-f192fde8a053.jpeg"  width="300"/>

<img src="https://user-images.githubusercontent.com/1218031/142572370-b824b516-a569-4277-b703-32f7edf507aa.PNG" width="300"/>

**Comment by 886merge** [comment]
merge886@gmail.com

**Comment by badcel** [comment]
It is still the same with 1.41

Now I get an additional red „an unknown error occurred“ message to the „try again“ button.

The error feels like the app is polling in the background as sometimes the error occurs then loads the page for 1 sec and then displays the error again (all this without user interaction).

If this is the case it would explain the rate limit violation.

**Comment by badcel** [comment, marked as answer]
Still happening with 1.42…

**Reply by badcel** [reply]
Fixed in version 1.47.1

**Comment by TTSxvilgax** [comment]
###############################################################################
# Set default behavior to automatically normalize line endings.
###############################################################################
* text=auto

###############################################################################
# Set default behavior for command prompt diff.
#
# This is need for earlier builds of msysgit that does not have it on by
# default for csharp files.
# Note: This is only used by command line
###############################################################################
#*.cs     diff=csharp

###############################################################################
# Set the merge driver for project and solution files
#
# Merging from the command prompt will add diff markers to the files if there
# are conflicts (Merging from VS is not affected by the settings below, in VS
# the diff markers are never inserted). Diff markers may cause the following 
# file extensions to fail to load in VS. An alternative would be to treat
# these files as binary and thus will always conflict and require user
# intervention with every merge. To do so, just uncomment the entries below
###############################################################################
#*.sln       merge=binary
#*.csproj    merge=binary
#*.vbproj    merge=binary
#*.vcxproj   merge=binary
#*.vcproj    merge=binary
#*.dbproj    merge=binary
#*.fsproj    merge=binary
#*.lsproj    merge=binary
#*.wixproj   merge=binary
#*.modelproj merge=binary
#*.sqlproj   merge=binary
#*.wwaproj   merge=binary

###############################################################################
# behavior for image files
#
# image files are treated as binary by default.
###############################################################################
#*.jpg   binary
#*.png   binary
#*.gif   binary

###############################################################################
# diff behavior for common document formats
# 
# Convert binary document formats to text before diffing them. This feature
# is only available from the command line. Turn it on by uncommenting the 
# entries below.
###############################################################################
#*.doc   diff=astextplain
#*.DOC   diff=astextplain
#*.docx  diff=astextplain
#*.DOCX  diff=astextplain
#*.dot   diff=astextplain
#*.DOT   diff=astextplain
#*.pdf   ![image](https://user-images.githubusercontent.com/96710718/147435296-f92548a0-bb98-42d5-9cc7-e009644652b4.jpeg)diff=astextplain
#*.PDF   diff=astextplain
#*.rtf   diff=astextplain
#*.RTF   diff=astextplain

**Comment by johnbent** [comment]
This is super-annoying. I run a volunteer non-profit webpage for the preservation of an endangered language. We have a volunteer group who consults about words which have not yet been documented. We used to use debugle for these discussions but debugle closed. No problem; github is awesome and free. But I wrote a script to automatically create issues in github from the debugle issues and I keep hitting the secondary rate limit. It shouldn't be a problem because I only need to create about 100 issues with 5 comments each. But I keep hitting this limit. It wouldn't be a problem if I could query the limit and then sleep accordingly. But when I try to query, all my rates look OK!!! Argh. :(

~/tekinged.com/scripts/debugle> curl -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $GH_OATH" https://api.github.com/rate_limit | grep remaining
"remaining": 4715,
"remaining": 30,
"remaining": 5000,
"remaining": 5000,
"remaining": 100,
"remaining": 1000,
"remaining": 10000,
"remaining": 15000,
"remaining": 100,
"remaining": 4715,
